src/client/game/scripting/function_tables.cpp

```diff
@@ -2984,7 +2984,7 @@ namespace scripting
 		{"primaryweapon", 0x33D},
```

Comment:
need to call this `_not_primaryweapon` if ur gonna have another token that's the exact same name

Reply:
I forgot about it, solved.